features/admin.connections.v1/api/connections.ts

```diff
@@ -859,6 +859,38 @@ export const getConnectedApps = (idpId: string): Promise<any> => {
         });
 };
 
+/**
+ * Hook to get connected apps of a local authenticator.
+ *
+ * @param authenticatorId - ID of the local authenticator.
+ * @param shouldFetch - Whether the request should be fetched.
+ * @returns Requested connected apps.
+ */
+export const useGetConnectedAppsOfAuthenticator = <Data = ConnectedAppsInterface, Error = RequestErrorInterface>(
```

Comment:
Lets add this api to a separate file.
Refer: https://github.com/wso2/identity-apps/tree/master/features/admin.actions.v1/api